core/src/trezor/ui/button.py

```diff
@@ -1,118 +1,28 @@
 from micropython import const
 
 from trezor import ui
-from trezor.ui import display, in_area
+from trezor.ui import in_area
+
+from .model import button as model
```

Comment:
perhaps worth only importing the `render_button` function?

Reply:
Fixed 2290a921f1c430d3742602b38a78be72fb23ceb1.